data/beaconrestapi/src/test/java/tech/pegasys/teku/beaconrestapi/handlers/v1/node/GetSyncingTest.java

```diff
@@ -45,6 +45,9 @@ public void shouldGetSyncingStatusSyncing() throws Exception {
         .isEqualTo(new GetSyncing.SyncStatusData(true, false, 7, 3));
   }
 
+  @Test
+  public void shouldGetSyncingStatusPreGenesis() {}
```

Comment:
Is this empty on purpose?

Reply:
left overs from another change earlier, oops!